CHANGELOG.md

```diff
@@ -2,9 +2,39 @@
 
 ## Table of Contents
 
+- [r0.2.0-rc.2](#r020-rc2)
 - [v0.2.0-rc.1](#v020-rc1)
 - [v0.1.0 - Initial version](#v010---initial-version)
 
+# r0.2.0-rc.2
+
+**This is the second release candidate version for "Identity And Consent Management" release 0.2.0**
+
+## Please note:
+
+* This is a bugfix release.
+
+### Main Changes
+* Fixed the release tag according to Release Management Working Group guidelines.
+* Fixed typo in `access_denied` error for authentication code flow.
+
+### Added
+* N/A
```

Comment:
```suggestion
* Replaced link into main branch of Identity and Consent Management within "Authorization and authentication" text template - need to be applied to all APIs who have copied the template previously
```

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/181/commits/ab5ec3c1bd4114c07a4f33e8bbd9c047ac1ebf7e

Reply:
@jpengar Upps, seems that I haven't put it into the right category. Would you mind to move it from "Added" to "Changed"?

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/181/commits/29279965f476d27be3c4b905f2d3b8e728602a31. My bad, I didn't notice it in my review either.